src/broker/broker.c

```diff
@@ -36,11 +36,11 @@
 #include <sys/syscall.h>
 #endif
 #if HAVE_VALGRIND
-# if HAVE_VALGRIND_H
-#  include <valgrind.h>
-# elif HAVE_VALGRIND_VALGRIND_H
-#  include <valgrind/valgrind.h>
-# endif
+#if HAVE_VALGRIND_H
```

Comment:
I prefer indented preprocessor macros as seen here. Could we consider adding `IndentPPDirectives: AfterHash`? Not important if nobody else cares...